crates/pallet-transaction-fees/src/lib.rs

```diff
@@ -254,6 +242,10 @@ where
         }
     }
 
+    pub fn transaction_byte_fee_storage_key() -> Vec<u8> {
```

Comment:
Similarly I really don't like that this abstraction is now leaking to the outside. Can you explain why it is needed and why can't it be achieved without making more implemenation details public?